lewis/adapters/epics.py

```diff
@@ -205,12 +207,22 @@ def start_server(self):
 
             The server does not process requests unless :meth:`handle` is called regularly.
         """
-        self._server = SimpleServer()
-        self._server.createPV(prefix=self._options.prefix,
-                              pvdb={k: v.config for k, v in self.pvs.items()})
-        self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+        if self._server is None:
+            self._server = SimpleServer()
+            self._server.createPV(prefix=self._options.prefix,
+                                  pvdb={k: v.config for k, v in self.pvs.items()})
+            self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+            self._driver.process_pv_updates(0.0, force=True)
+
+            self._last_update = datetime.now()
```

Comment:
While we're mucking around in here... `_last_update` could use some polish. 

It's defined outside of `__init__`. Also, it's only really user inside `PropertyExposingDriver.process_pv_updates()`. But defined here as a member of `EpicsAdapter`. It's all a bit awkward.